ccm_web/server/src/canvas/canvas.interfaces.ts

```diff
@@ -111,7 +134,7 @@ export interface CanvasEnrollment {
   course_id: number
   course_section_id: number
   user_id: number
-  type: UserEnrollmentType
+  type: ClientEnrollmentType
```

Comment:
I actually think this needs to stay the same: `CanvasEnrollment` should be the data we expect back from Canvas, and it won't use the role representations we use in the frontend.

Reply:
I fixed this.